src/api/app/models/staging/staged_requests.rb

```diff
@@ -91,6 +91,9 @@ def link_package(bs_request_action)
     source_package = Package.get_by_project_and_name!(bs_request_action.source_project,
                                                       bs_request_action.source_package)
 
+    target_package = Package.get_by_project_and_name!(bs_request_action.target_project,
```

Comment:
This will throw an exception for submissions of packages that would be new in `target_project`. It shouldn't.